prawcore/auth.py

```diff
@@ -48,8 +48,9 @@ def authorize_url(self, duration, scopes, state, implicit=False):
             ``temporary`` can be specified if ``implicit`` is set to ``True``.
         :param scopes: A list of OAuth scopes to request authorization for.
         :param state: A string that will be reflected in the callback to
-            ``redirect_uri``. This value should be temporarily unique to the client for
-            whom the URL was generated for.
+            ``redirect_uri``. Elements must be printable ASCII characters in the range
+            0x20 through 0x7E inclusive. This value should be temporarily unique to the
```

Comment:
Could we include a regex pattern instead of the hex range for what is allowed?

Reply:
I think the hex values are suitable. One would have to look up an ASCII table regardless to know what characters are between the start and end characters, and `[ -~]` I feel like conveys less information (if it even works properly) than using the ASCII numbers.